README.md

```diff
@@ -157,42 +157,45 @@ The simplest way to get started making accurate rips is:
 
 ## Configuration file documentation
 
-The configuration file is stored according to the [XDG Base Directory Specification](
-http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html)
-when possible.
+The configuration file is stored in  
+`$XDG_CONFIG_HOME/whipper/whipper.conf` or  
+`$HOME/.config/whipper/whipper.conf` if `$XDG_CONFIG_HOME` is undefined.
```

Comment:
I would probably separate the two options a bit stronger. Maybe a comma (` , `) or (en‐)hyphen (` – `) before the `or`?

Reply:
Yeah, exactly.